Library/Homebrew/test/formulary_spec.rb

```diff
@@ -496,25 +498,16 @@ def formula_json_contents(extra_items = {})
           FileUtils.rm_rf foo_tap.path
         end
 
-        it "returns the core formula if the short name clashes with a tap migration rename" do
-          (foo_tap.path/"tap_migrations.json").write <<~JSON
-            { "#{formula_name}": "homebrew/core/#{formula_name}-v2" }
-          JSON
-
-          expect(described_class::FromNameLoader.try_new(formula_name))
-            .to be_a(described_class::FromNameLoader)
-            .and have_attributes(name: formula_name)
-        end
```

Comment:
This test is not so relevant anymore. With the API, short names will be evaluated before ever hitting the name loader.